src/main/java/com/zepben/testutils/vertx/DeployRestVerticleHelper.java

```diff
@@ -40,13 +40,13 @@ public DeployRestVerticleHelper(Class<?> verticleClass, JsonObject config) {
             vertx = Vertx.vertx();
             DeploymentOptions options = new DeploymentOptions().setConfig(config);
             vertx.deployVerticle(verticleClass.getName(),
-                options,
-                ar -> {
-                    if (ar.succeeded())
-                        future.complete();
-                    else
-                        future.fail(ar.cause());
-                });
+                    options,
+                    ar -> {
+                        if (ar.succeeded())
+                            future.complete();
+                        else
+                            future.fail(ar.cause());
+                    });
```

Comment:
Extra indentation on continuation reveals that the settings in your IDE are set to the default 8 rather than the prefered 4. It also revealed the settings for the project were not added to git, so I have fixed that.